backend/kesaseteli/applications/exporters/excel_exporter.py

```diff
@@ -0,0 +1,228 @@
+        ["application__company__company_form"],
+        15,
+        "#E7E3F9",
+    ),
+    (_("Työnantaja"), "%s", ["application__company__name"], 30, "#E7E3F9"),
+    (_("Y-tunnus"), "%s", ["application__company__business_id"], 15, "#E7E3F9"),
+    (
+        _("Työnantajan lähiosoite"),
+        "%s",
+        ["application__company__street_address"],
+        30,
+        "#E7E3F9",
+    ),
+    (
+        _("Työnantajan postinumero"),
+        "%s",
+        ["application__company__postcode"],
+        15,
+        "#E7E3F9",
+    ),
+    (_("Työnantajan kunta"), "%s", ["application__company__city"], 15, "#E7E3F9"),
+    (_("Yhdyshenkilö"), "%s", ["application__contact_person_name"], 30, "#E7E3F9"),
+    (
+        _("Yhdyshenkilön sähköposti"),
+        "%s",
+        ["application__contact_person_email"],
+        30,
+        "#E7E3F9",
+    ),
+    (
+        _("Yhdyshenkilön Puhelin"),
+        "%s",
+        ["application__contact_person_phone_number"],
+        30,
+        "#E7E3F9",
+    ),
+    (_("Yrityksen toimiala"), "%s", ["application__company__industry"], 30, "#E7E3F9"),
+    ("", "", [], 5, "#7F7F7F"),
+    (
+        _("Työn suorituspaikan postinumero"),
+        "%s",
+        ["employment_postcode"],
+        15,
+        "#F7DAE3",
+    ),
+    (
+        _("Työsuhteen kesto"),
+        "%s-%s",
+        ["employment_start_date", "employment_end_date"],
+        30,
+        "#F7DAE3",
+    ),
+    (_("Työtunnit"), "%s", ["employment_work_hours"], 15, "#F7DAE3"),
+    (_("Maksettu palkka"), "%s", ["employment_salary_paid"], 15, "#F7DAE3"),
+    (_("Muut edut"), "%s", [""], 15, "#F7DAE3"),
+    (_("Raporttiin luokittelu"), "%s", [""], 15, "#F7DAE3"),
+    (_("Työtehtävät"), "%s", ["employment_description"], 30, "#F7DAE3"),
+    (
+        _("Olisitko palkannut?"),
+        "%s",
+        ["hired_without_voucher_assessment"],
+        15,
+        "#F7DAE3",
+    ),
+    (_("Työnantajan kokemus"), "%s", [""], 30, "#F7DAE3"),
+    (_("Muuta"), "%s", [""], 30, "#F7DAE3"),
+)
+
+
+def get_xlsx_list() -> list:
+    if os.path.exists(settings.EXCEL_ROOT):
+        return sorted(os.listdir(settings.EXCEL_ROOT), reverse=True)
+    return []
+
+
+def get_filename_with_index(file_index: int) -> str:
+    xlsx_list = get_xlsx_list()
+    try:
+        filename = xlsx_list[file_index]
+    except IndexError:
+        return ""
+    return filename
+
+
+def get_xlsx_filename() -> str:
+    """
+    Get the name of the excel file. Example filename:
+    kesasetelihakemukset_2021-01-01_23-59-59.xlsx
+    """
+    local_datetime_now_as_str = timezone.localtime(timezone.now()).strftime(
+        "%Y-%m-%d_%H-%M-%S"
+    )
+    filename = f"kesasetelihakemukset_{local_datetime_now_as_str}.xlsx"
+    return filename
+
+
+def get_xlsx_filepath() -> str:
+    """
+    Get the path for the excel and create the folders if they don't exist yet.
+    """
+    if not os.path.exists(settings.EXCEL_ROOT):
+        os.makedirs(settings.EXCEL_ROOT)
```

Comment:
what group/global permissions does the folder get? Maybe limit to the app user only?

Reply:
I left out this feature for now.